src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -316,15 +330,22 @@ def open_pr_comment_workflow(pr_id: int) -> None:
     client = installation.get_client()
 
     # CREATING THE COMMENT
-    if not safe_for_comment(gh_client=client, repository=repo, pull_request=pull_request):
+    logger.info("github.open_pr_comment.check_safe_for_comment")
```

Comment:
Do we need this logger b/c `safe_for_comment` has one [here](https://github.com/getsentry/sentry/pull/61981/files#diff-ac61a65f5e0d54d5711c6fbf9a39731eb359414ef36809e9938aa8a18a8286e9R139) at the start

Reply:
no, i can remove

Reply:
i can do this in the follow up